ProteoformSuiteInternal/Proteoform.cs

```diff
@@ -212,6 +213,7 @@ public List<ExperimentalProteoform> identify_connected_experimentals(List<PtmSet
                         rank_sum += Sweet.lollipop.neucode_labeled ?
                             Sweet.lollipop.mod_rank_third_quartile :   //in neucode-labeled data, fewer missed monoisotopics - don't prioritize
                             1; //in label-free, more missed monoisotoipcs, should prioritize (set to same priority as variable modification)
+                        rank_sum -= additional_ptm_penalty;
```

Comment:
I don't want missed mono to be counted as an additional ptm

I'm thinking of reworking missed monos as a param for allowing up to X number and then treating them as unmodified because they're so common in unlabeled. I saw cases where a theoretical gets matched with unmodified with mass error 0.04 or something while the missed mono would be spot on (1.002 mass diff)